setup.py

```diff
@@ -1,65 +1,72 @@
-      'ext_modules': [Extension('scout_apm.core.objtrace', ['src/scout_apm/core/ext/objtrace.c'])],
-      'entry_points': {
-          'console_scripts': [
-              'core-agent-manager = scout_apm.core.cli.core_agent_manager:main'
-          ]
-      },
-      'install_requires': ['psutil', 'PyYAML', 'requests'],
-      'keywords': 'apm performance monitoring development',
-      'classifiers': [
-        'Development Status :: 5 - Production/Stable',
-        'Intended Audience :: Developers',
-        'Topic :: System :: Monitoring',
-        'License :: OSI Approved :: MIT License',
-        'Operating System :: MacOS',
-        'Operating System :: POSIX',
-        'Operating System :: POSIX :: Linux',
-        'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
-        'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: 3.6',
-        'Programming Language :: Python :: 3.7',
-      ]}
+    "name": "scout_apm",
+    "version": "1.3.8",
+    "description": "Scout Application Performance Monitoring Agent",
+    "long_description": long_description,
+    "long_description_content_type": "text/markdown",
+    "url": "https://github.com/scoutapp/scout_apm_python",
+    "author": "Scout",
+    "author_email": "support@scoutapp.com",
+    "license": "MIT",
+    "zip_safe": False,
+    "python_requires": ">=3.4, <4",
+    "packages": find_packages("src"),
+    "package_dir": {"": "src"},
+    "py_modules": [os.splitext(os.basename(path))[0] for path in glob("src/*.py")],
+    "ext_modules": [
+        Extension("scout_apm.core.objtrace", ["src/scout_apm/core/ext/objtrace.c"])
+    ],
+    "entry_points": {
+        "console_scripts": [
+            "core-agent-manager = scout_apm.core.cli.core_agent_manager:main"
+        ]
+    },
+    "install_requires": ["psutil", "PyYAML", "requests"],
+    "keywords": "apm performance monitoring development",
+    "classifiers": [
+        "Development Status :: 5 - Production/Stable",
+        "Intended Audience :: Developers",
+        "Topic :: System :: Monitoring",
+        "License :: OSI Approved :: MIT License",
+        "Operating System :: MacOS",
+        "Operating System :: POSIX",
+        "Operating System :: POSIX :: Linux",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.4",
+        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+    ],
+}
 
 if sys.version_info <= (3, 0):
     compile_extensions = False
 
-if sys.platform.startswith('java'):
+if sys.platform.startswith("java"):
     compile_extensions = False
 
-if '__pypy__' in sys.builtin_module_names:
+if "__pypy__" in sys.builtin_module_names:
     compile_extensions = False
 
-if compile_extensions is not True:
-    del setup_args['ext_modules']
+if not compile_extensions:
```

Comment:
These comparisons are not equivalent. `if compile_extensions is not True` is looking for exactly `True` and not just any truthy value. I generally prefer the explicit `is not True` in order to prevent bugs where we have some truthy value unintentionally.

Reply:
Ha, I know you know the difference :)

I suppose there are quite a few things I don't agree with in Python, this being one of them. I don't mind using `if not compile_extensions` style to check for existence/presence, but I dislike it when I'm actually checking for a specific value. If it's a code smell then we can change it as you have.

`and doesn't cause bugs.` 🤞Let's hope! We've seen it in Ruby ;)